ui/src/playlist/PlaylistSongs.js

```diff
@@ -172,20 +174,26 @@ const PlaylistSongs = ({ playlistId, readOnly, ...props }) => {
         </Card>
       </div>
       <AddToPlaylistDialog />
+      {React.cloneElement(props.pagination, listContext)}
     </>
   )
 }
 
 const SanitizedPlaylistSongs = (props) => {
-  const { loaded, loading, total, ...rest } = useListContext(props)
+  const { loaded, ...rest } = props
```

Comment:
This (use of `useListContext` hook) took me a white to get correctly, with the fewer number of redraws as possible. I'll have to do some testing on this